### ISSUE TITLE
Error in maven plugin generating client code from introspection result (Version: 4.1.2)

### ISSUE THREAD

**Issue by konjp** [kobylynskyi/graphql-java-codegen#535]
## Issue Description

When trying to generate client code from introspection result exception is thrown:
`java.lang.IllegalArgumentException: Either graphql schema path or introspection result path should be supplied
    at com.kobylynskyi.graphql.codegen.GraphQLCodegen.validateConfigs (GraphQLCodegen.java:208)
    at com.kobylynskyi.graphql.codegen.GraphQLCodegen.<init> (GraphQLCodegen.java:99)
    at com.kobylynskyi.graphql.codegen.GraphQLCodegen.<init> (GraphQLCodegen.java:79)
    at com.kobylynskyi.graphql.codegen.java.JavaGraphQLCodegen.<init> (JavaGraphQLCodegen.java:27)
    at io.github.kobylynskyi.graphql.codegen.GraphQLCodegenMojo.instantiateCodegen (GraphQLCodegenMojo.java:258)
    at io.github.kobylynskyi.graphql.codegen.GraphQLCodegenMojo.execute (GraphQLCodegenMojo.java:248)`

This is happing when only _graphqlQueryIntrospectionResultPath_ is supplied without _graphqlSchemas_ in pom.xml. This should be valid scenario, because code should be possible to generate only from introspection result.

## Steps to Reproduce
*List in detail the exact steps to reproduce the unexpected behavior.*

## Expected Result
*Explain in detail what behavior you expected to happen.*

## Actual Result
I think this part of code:

`if (!Utils.isEmpty(schemas) && introspectionResult != null ||
                (Utils.isEmpty(schemas) && introspectionResult == null)) {
            // either schemas or introspection result should be provided
            throw new IllegalArgumentException("Either graphql schema path or introspection result path should be supplied");
        }`

should be replaced with

`if ( Utils.isEmpty(schemas) && Utils.isBlank(introspectionResult) ) {
            // either schemas or introspection result should be provided
            throw new IllegalArgumentException("Either graphql schema path or introspection result path should be supplied");
        }`

## Your Environment and Setup

* **graphql-java-codegen**: *4.1.2*
* **Build tool**: *Maven*
* **Java tool**: *11.0.8*
* **Mapping Config**: *E.g.:* 

```xml
<configuration>
    <outputDir>${project.build.directory}/generated-sources/graphql</outputDir>
    <packageName>io.github.kobylynskyi.graphql.model</packageName>
</configuration>
```

**Comment by kobylynskyi** [comment]
@konjp, thanks for reporting this.
I've fixed it in 4.1.3 and will release the library and maven/gradle plugins this week. Stay tuned!

**Comment by kobylynskyi** [comment]
@konjp, was released in [4.1.3](https://github.com/kobylynskyi/graphql-java-codegen/releases/tag/v4.1.3)
